main/index.js

```diff
@@ -68,10 +68,14 @@ if (process.platform === 'win32') {
 }
 
 // Only one instance can run at a time
-if (!app.requestSingleInstanceLock() && !inTest) {
+if (!inTest && !app.requestSingleInstanceLock()) {
   app.quit()
 }
 
+app.on('second-instance', () => {
```

Comment:
_Nitpick: feel free to ignore._

I would add a comment to explain the UX flow we are implementing here. E.g.

```ts
// When the user attempts to start the app from the menu because 
// they don't notice the Station icon in the tray, then help 
// them finding the Station by showing our main window.
app.on('second-instance', () => {
```

After I wrote this comment, I realised it would be even nicer if the main window could also show a dialog explaining to the user that the Station has been running in the background and that they can always find it in the Tray (macOS menu bar). That's probably out of the scope of this PR, though.

Reply:
great suggestions, added docs and opened https://github.com/filecoin-station/filecoin-station/issues/282